spec/models/reviewable_note_spec.rb

```diff
@@ -0,0 +1,123 @@
+        note = build(:reviewable_note, content: "", reviewable: reviewable, user: admin)
+        expect(note).not_to be_valid
+        expect(note.errors[:content]).to include("can't be blank")
+      end
+
+      it "is invalid with whitespace-only content" do
+        note = build(:reviewable_note, content: "   ", reviewable: reviewable, user: admin)
+        expect(note).not_to be_valid
+        expect(note.errors[:content]).to include("can't be blank")
+      end
+    end
+  end
+
+  describe "scopes" do
+    let!(:old_note) do
+      Fabricate(:reviewable_note, reviewable: reviewable, user: admin, created_at: 2.days.ago)
+    end
+    let!(:new_note) do
+      Fabricate(:reviewable_note, reviewable: reviewable, user: moderator, created_at: 1.day.ago)
+    end
+
+    describe ".ordered" do
+      it "returns notes ordered by creation date ascending" do
+        notes = ReviewableNote.ordered
+        expect(notes.first).to eq(old_note)
+        expect(notes.last).to eq(new_note)
+      end
+    end
+  end
+
+  describe "staff permission check" do
+    it "allows staff users to create notes (permission checked in controller)" do
+      admin_note = build(:reviewable_note, reviewable: reviewable, user: admin)
+      expect(admin_note.save).to be_truthy
+
+      moderator_note = build(:reviewable_note, reviewable: reviewable, user: moderator)
+      expect(moderator_note.save).to be_truthy
+    end
+
+    it "allows regular users to create notes at model level (permission enforced in controller)" do
+      # Model doesn't enforce staff requirement - that's done in controller
+      note = build(:reviewable_note, reviewable: reviewable, user: user)
+      expect(note.save).to be_truthy
+    end
+
+    it "prevents creation when user is nil" do
+      note = build(:reviewable_note, reviewable: reviewable, user: nil)
+      expect(note.save).to be_falsey
+      expect(note.errors[:user_id]).to include("can't be blank")
+    end
+  end
+
+  describe "factory" do
+    it "creates a valid reviewable note" do
+      note = Fabricate(:reviewable_note, reviewable: reviewable, user: admin)
+      expect(note).to be_valid
+      expect(note.content).to be_present
+      expect(note.reviewable).to eq(reviewable)
+      expect(note.user).to eq(admin)
+    end
+  end
+
+  describe "content handling" do
+    it "preserves line breaks in content" do
+      content_with_breaks = "Line 1\nLine 2\n\nLine 4"
+      note =
+        Fabricate(
+          :reviewable_note,
+          content: content_with_breaks,
+          reviewable: reviewable,
+          user: admin,
+        )
+      expect(note.content).to eq(content_with_breaks)
+    end
+
+    it "handles unicode characters" do
+      unicode_content = "This is a note with emojis 🎉 and accented characters café"
+      note =
+        Fabricate(:reviewable_note, content: unicode_content, reviewable: reviewable, user: admin)
+      expect(note.content).to eq(unicode_content)
+    end
+  end
```

Comment:
I do wonder if we need these tests here since we are just relying on postgres' text column and there isn't any logic unique to the application that would result in a regression here.